.github/workflows/intelligence-docs.yml

```diff
@@ -23,20 +23,28 @@ env:
 
 jobs:
   build_and_deploy:
-    runs-on: ubuntu-22.04
+    runs-on: macos-14
```

Comment:
Can we build the swift docs on a separate job? The macos machines are more expensive and I would prefer to keep building the normal docs on ubuntu - which is basically for free.